plugins/formal-verification/testData/diagnostics/no_contracts/classes_getters.kt

```diff
@@ -10,13 +10,6 @@ class IntWrapper(val n: Int) {
 
 class IntWrapperContainer(val i: IntWrapper)
 
-fun <!VIPER_TEXT!>createFoo<!>() {
-    val f: Foo = Foo(10, 20)
-    val fa = f.a
-    val fb = f.b
-}
```

Comment:
I think this should be kept to test getting field value for backing fields, right?

Reply:
This exact code is present in `classes.kt` already, so there's no point having it here as well

Reply:
Ah makes sense